### PR TITLE
[#8528] Instructor: view results: make the explanation of the view more informative

### LINKED ISSUE #8528: Instructor: view results: make the explanation of the view more informative
Current:
image

Suggestion:
1. Use an ascii arrow `→` rather than `>` (both in this view, and in the drop down menu)
2. Give a more precise tooltip based on the value e.g., `Group responses by giver, then by recipient, and then by question`

### PR DESCRIPTION
**Pull request by J-Eisen**
Fixes #8528 

**Outline of Solution**

- Replaced ">" and "&gt;" with "&#8594;" and "→" where needed
- Removed div with "View sections in different formats" from feedback results view (NOT selection)
- Added more specific div wrappers around "Group by -" text
- Updated InstructorFeedbackResultsPage.java to test for new text

### CODE REVIEW

File: src/main/webapp/WEB-INF/tags/instructor/results/bySectionPanels.tag [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <%@ tag trimDirectiveWhitespaces="true" %>
-<%@ tag description="instructorFeedbackResults - by Giver > Question > Recipient or Recipient > Question > Giver" pageEncoding="UTF-8" %>
+<%@ tag description="instructorFeedbackResults - by Giver &#8594; Question &#8594; Recipient or Recipient &#8594; Question &#8594; Giver" pageEncoding="UTF-8" %>
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/results/bySectionPanels.tag]: Can use the → symbol directly as UTF-8 is supported. Example of a UTF-8 character in the current codebase: https://github.com/TEAMMATES/teammates/blob/1bc5f6589a9003ceb9c2c8f2f3c22d6bdaa0d173/src/main/webapp/WEB-INF/tags/instructor/course/studentTableModal.tag#L13

Comment (J-Eisen) [reply src/main/webapp/WEB-INF/tags/instructor/results/bySectionPanels.tag]: Thanks! I must have missed the other UTF-8 characters and was being (unnecessarily) cautious.

**Comment by joanneong** [comment]
@J-Eisen Did you try to run the test locally? I just did it, and with GodMode the changes are as per expected. How are you running GodMode? Did you modify the correct property in `test.properties`?

**Comment by J-Eisen** [comment]
@joanneong I've been changing the test.godmode.enabled on line 46 to "true," running the tests (local & ciTests), then rerunning them after changing it back to "false." (And, of course, saving test.properties in between.) Just reran it again using local tests in terminal. Everything is passing but no files have changed. :-\

Sure! Sorry about that.

**Comment by J-Eisen** [comment]
@joanneong I've been rerunning using the whole suite followed by failedTests, which should be running InstructorFeedbackResultsPageUiTest.java, if I understand correctly.

I've just given running InstructorFeedbackResultsPageUiTest a try. It looks like the right files have been updated! Hopefully, it passes the build.

Could you explain why running the specific test worked when the whole suite didn't? I don't understand why the ciTests and localTests didn't create the proper changes. :-/

**Comment by joanneong** [comment]
@J-Eisen How did you run the test suite? Using a button in Intellij (or Eclipse)? Because I think that it should work as well :P

**Comment by J-Eisen** [comment]
Hi @whipermr5! Passing the checks and ready for you to take a look. :)

**Comment by joanneong** [comment]
@J-Eisen Note that there may be some delays for the review because we just ended our school semester - but before he reviews it, can you try to resolve the conflicts again first? :)

**Comment by J-Eisen** [comment]
@joanneong @whipermr5 Ah! Forgot it was exam season. (And didn't notice the conflicts.) I'll get right on that. Hope your exams are going well!

**Pull request closed without merging**